libs/common/HashSlotUtils.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Garnet.common
+{
+    public static unsafe class HashSlotUtils
+    {
+        /// <summary>
+        /// This table is based on the CRC-16-CCITT polynomial (0x1021)
```

Comment:
I had it wrong when I wrote this comment in https://github.com/microsoft/garnet/pull/198
```suggestion
        /// This table is based on the CRC-16-XMODEM polynomial (0x1021)
```